use/wordcount-example.md

```diff
@@ -0,0 +1,334 @@
+
+The WordCount examples demonstrate how to set up a processing pipeline that can read text, tokenize the text lines into individual words, and perform a frequency count on each of those words. The Beam SDKs contain a series of these four successively more detailed WordCount examples that build on each other. The input text for all the examples is a set of Shakespeare's texts.
+
+Each WordCount example introduces different concepts in the Dataflow SDKs. Begin by understanding Minimal WordCount, the simplest of the examples. Once you feel comfortable with the basic principles in building a pipeline, continue on to learn more concepts in the other examples.
+
+* **Minimal WordCount** demonstrates the basic principles involved in building a Dataflow pipeline.
+* **WordCount** introduces some of the more common best practices in creating re-usable and maintainable pipelines.
+* **Debugging WordCount** introduces logging and debugging practices.
+* **Windowed WordCount** demonstrates how you can use Dataflow's programming model to handle both bounded and unbounded datasets. Note: Windowed WordCount is not yet available in the Python SDK.
+
+## MinimalWordCount
+
+Minimal WordCount demonstrates a simple pipeline that can read a block of text from Google Cloud Storage, apply transforms to tokenize and count the words, and write the data to an output text file. This example hard-codes the locations for its input and output files and doesn't perform any error checking; it is intended to only show you the "bare bones" of creating a Beam pipeline. In later examples, we will parameterize the pipeline's input and output sources and show other best practices.
+
+To run this example, follow the instructions in the [Beam Examples README](https://github.com/apache/incubator-beam/blob/master/examples/java/README.md#building-and-running). To view the full code, see **[MinimalWordCount](https://github.com/apache/incubator-beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/MinimalWordCount.java).**
+
+**Key Concepts:**
+
+* Creating the Pipeline
+* Applying transforms to the Pipeline
+* Reading input (in this example: reading text files)
+* Applying ParDo transforms
+* Applying SDK-provided transforms (in this example: Count)
+* Writing output (in this example: writing to a text file)
+* Running the Pipeline
+
+The following sections explain these concepts in detail along with excerpts of the relevant code from the Minimal WordCount pipeline.
+
+### Creating the Pipeline
+
+The first step in creating a Beam pipeline is to create a Pipeline Options object. This object lets us set various options for our pipeline, such as the pipeline runner that will execute our pipeline, the ID of our project, and the staging location for the pipeline to store its files (used to make your jars accessible in the Cloud). In this example we set these options programmatically, but more often command-line arguments are used to set Pipeline options.
```

Comment:
Id, staging location, etc are Dataflow specific. Perhaps use input file and "any runner-specific configuration required by the chosen runner" ?